digdag-docs/src/command_reference.rst

```diff
@@ -106,6 +106,11 @@ Options:
 
   Example: ``--max-task-threads 5``
 
+:command:`-O, --task-log DIR`
+  Store task logs to this directory.
+
+  Example: ``--task-log digdag.log``
```

Comment:
`digdag.log` looks like file name. we'd better to change it with directory-like name. for example, `log/tasks` or something. what do you think?

Reply:
@muga Thank you for your pointing out.
I agree with you and fix it.
If there are something other issues, please let me know.

Best regards,
  You Yamagata